### DISCUSSION TITLE
[resolu] 09. Ne pas afficher les flux des réseaux sociaux

### DISCUSSION THREAD

**Discussion by hrenaud** [cnumr/best-practices-wordpress#13] [Bonnes pratiques] [answered]
@dNicolle 

De par mes échanges sur le sujet avec des UX, il me semble que l'on pourrait rappeler que le but d'animer un RS est de faire découvrir ou d'annoncer de nouvelles publications de son site, donc de le **faire venir sur son site**.
Hors afficher ses flux RS, aurait **tendance à faire repartir ses utilisateurs vers les RS ce qui est contre-productif**.

De plus, si l'on a plusieurs fils RS + son fil d'actualité d'affiché sur sa home (par exemple), on se retrouve à avoir 2, 3 fois les mêmes articles, ce qui a plusieurs conséquences : 
- lourdeur de la page,
- duplication du contenus et des visuels
- l'utilisateur ne sait plus où cliquer et se perd dans la navigation (site → RS → site...).

**Comment by dNicolle** [comment]
Je pensais que les éléments indiqués permettaient de l'indiquer, mais, cela sera plus clair de l'écrire oui.

Je propose de remplacer 
```
Si intégrer les flux de vos réseaux sociaux sur votre site internet semble intéressant car il l'enrichi, c'est en fait contre productif pour plusieurs raisons 
```

par 

```
Animer les réseaux sociaux a pour objectif de faire venir les internautes sur votre site internet.
Intégrer les flux de vos réseaux sociaux sur votre site internet peut sembler pertinent pour le dynamiser et l'enrichir, c'est en fait contre productif pour plusieurs raisons :
```

Ensuite, je garde les 3 points cités, que je modifie ainsi
```
 - La cible n'est pas la même : En effet, vous ne touchez pas les mêmes personnes lorsque vous publiez des informations sur votre site ou sur un  réseau social. Le but étant de toucher plus de personne avec du contenu approprié
 - Votre site charge des éléments techniques qui viennent du réseau social et l'alourdissent
 - Vous risquez de présenter plusieurs fois les mêmes éléments. Votre article WordPress est publié sur un réseau social qui lui même est affiché sur votre site : vos visiteurs seront perdus (site > réseau sociaus > site...)
 - Si vous affichez plusieurs flux de réseaux sociaux avec les mêmes éléments venant du site, vos contenus seront d'autant plus mutliples et difficilement compréhensibles par l'internaute

```

@hrenaud ça répondrais à ton point de vigileance ?

**Reply by hrenaud** [reply]
Parfait.

Penses-tu aussi qu'il faille alerter sur le point que si le RS changent ses API, les plugins ne fonctionneront peut-être plus (si plus maintenus) ou pourrait mettre du temps à s'adapter et faire des bugs ?

**Comment by dNicolle** [comment, marked as answer]
Modification faite
